src/utils/editor/source-documents.js

```diff
@@ -24,18 +26,31 @@ function clearDocuments() {
 }
 
 function resetLineNumberFormat(editor: Object) {
+  if (!wasmLineNumberFormatters.has(editor)) {
+    return;
+  }
   let cm = editor.codeMirror;
   cm.setOption("lineNumberFormatter", number => number);
+  SourceEditorUtils.resizeBreakpointGutter(cm);
```

Comment:
I think we cannot remove this line since we (I) cannot guarantee editors state before updateLineNumberFormat is called.